app/domain/errors.rb

```diff
@@ -126,24 +126,32 @@ module AuthnOidc
           code: "CONJ00012E"
         )
 
-        IdTokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
-          msg: "Field '{0-field-name}' not found or empty in ID Token. This field is defined in the id-token-user-property variable.",
-          code: "CONJ00013E"
-        )
+      end
 
-        TokenVerifyFailed = ::Util::TrackableErrorClass.new(
-          msg: "Token verification failed (3rdPartyError ='{0}')",
-          code: "CONJ00015E"
+      module Jwt
+
+        TokenExpired = ::Util::TrackableErrorClass.new(
+          msg: "Token expired",
+          code: "CONJ00016E"
         )
 
         TokenDecodeFailed = ::Util::TrackableErrorClass.new(
           msg: "Token decode failed (3rdPartyError ='{0}')",
           code: "CONJ00035E"
         )
 
-        IdTokenExpired = ::Util::TrackableErrorClass.new(
-          msg: "Token expired",
-          code: "CONJ00016E"
+        TokenVerifyFailed = ::Util::TrackableErrorClass.new(
+          msg: "Token verification failed (3rdPartyError ='{0}')",
+          code: "CONJ00015E"
+        )
+
+      end
+
+      module AuthnOidc
+
+        IdTokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+          msg: "Field '{0-field-name}' not found or empty in ID Token. This field is defined in the id-token-user-property variable.",
```

Comment:
`id-token-user-property` might be changed so I don't necessary like us hardcoding it here just in case this might change in the future

Reply:
It is hard-coded in other places too. Let's not go to this rabbit hole in this PR